[2022-10-31 20:57:56,758] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: user_processing.create_table 2022-10-31T20:57:55.134498+00:00 [queued]>
[2022-10-31 20:57:56,774] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: user_processing.create_table 2022-10-31T20:57:55.134498+00:00 [queued]>
[2022-10-31 20:57:56,775] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-10-31 20:57:56,775] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-10-31 20:57:56,775] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-10-31 20:57:56,784] {taskinstance.py:1063} INFO - Executing <Task(PostgresOperator): create_table> on 2022-10-31T20:57:55.134498+00:00
[2022-10-31 20:57:56,789] {standard_task_runner.py:52} INFO - Started process 104 to run task
[2022-10-31 20:57:56,791] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'create_table', '2022-10-31T20:57:55.134498+00:00', '--job-id', '24', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp9w02sun9', '--error-file', '/tmp/tmprwsetlp2']
[2022-10-31 20:57:56,792] {standard_task_runner.py:77} INFO - Job 24: Subtask create_table
[2022-10-31 20:57:56,823] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.create_table 2022-10-31T20:57:55.134498+00:00 [running]> on host 8311e947f87b
[2022-10-31 20:57:56,855] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-10-31T20:57:55.134498+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-31T20:57:55.134498+00:00
[2022-10-31 20:57:56,855] {postgres.py:69} INFO - Executing: 
                CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
                );
            
[2022-10-31 20:57:56,863] {base.py:74} INFO - Using connection to: id: postgres. Host: postgres, Port: 5432, Schema: , Login: airflow, Password: XXXXXXXX, extra: None
[2022-10-31 20:57:56,866] {dbapi.py:180} INFO - Running statement: 
                CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
                );
            , parameters: None
[2022-10-31 20:57:56,866] {dbapi.py:186} INFO - Rows affected: -1
[2022-10-31 20:57:56,867] {postgres.py:73} INFO - NOTICE:  relation "users" already exists, skipping

[2022-10-31 20:57:56,874] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20221031T205755, start_date=20221031T205756, end_date=20221031T205756
[2022-10-31 20:57:56,900] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-10-31 20:57:56,922] {local_task_job.py:146} INFO - Task exited with return code 0
